.gitignore

```diff
@@ -35,9 +35,6 @@ Thumbs.db
 # Local Netlify folder
 .netlify
 
-# For proper npm login
-.npmrc
```

Comment:
Just wondering what was it for.

Reply:
I suppose that previous maintainers used `.npmrc` to store their auth information to release Boosted package.